cmd/template/cluster/provider/templates/vsphere/vsphere_cluster.yaml.tmpl

```diff
@@ -0,0 +1,19 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
+kind: VSphereCluster
+metadata:
+  labels:
+    "release.giantswarm.io/version": "{{ .Version }}"
+    "giantswarm.io/cluster": "{{ .Name }}"
+    "cluster.x-k8s.io/cluster-name": "{{ .Name }}"
+    "giantswarm.io/organization": "{{ .Organization }}"
+  name: {{ .Name }}
+  namespace: {{ .Namespace }}
+spec:
+  controlPlaneEndpoint:
+    host: 10.0.6.191 #CHANGE HERE
+    port: 6443
+  identityRef:
+    kind: Secret
+    name: obojan02
```

Comment:
And this?

Reply:
Forgot to discuss this today - this secret needs to be shared IIRC. It contains an identity which is either unique per installation or organization. Not sure which.